bns/getKoboData.js

```diff
@@ -1,16 +1,19 @@
-// Here we fetch submissions for all "Deployed" forms in GoogleSheet 
+// Here we fetch submissions for all "Deployed" forms in GoogleSheet
 // NOTE: See linked job "[BNS-1A] 1.Get FormsList (Ongoing)" for cursor & GoogleSheet query logic
 //**********************************************************//
+
 fn(state => {
-  console.log('surveys ::', JSON.stringify(state.data.surveys, null, 2)); 
-  return state; 
-})
+  state.surveySubmissions = state.surveySubmissions || [];
+  state.errors = state.errors || [];
```

Comment:
If we have other errors from previous run they will be carried into this step and displayed as new errors which is not accurate 

`state.errors = []` makes sure we are only collecting errors for this step. Is it not working ?
@hunterachieng